package.json

```diff
@@ -21,10 +31,10 @@
     "babel-cli": "^6.24.1",
     "babel-core": "^6.24.1",
     "babel-preset-es2015": "^6.24.1",
-    "babel-preset-stage-1": "^6.24.1"
+    "babel-preset-stage-1": "^6.24.1",
+    "postinstall-build": "^5.0.1"
```

Comment:
I agree, one thing at time. Also, I need to further investigate the issue on my side.